record/record_test.go

```diff
@@ -332,22 +332,52 @@ func writeRecords(records [][]byte) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
-func TestBasicRecover(t *testing.T) {
-	records := [][]byte{
-		[]byte(strings.Repeat("a", blockSize-headerSize)),
-		[]byte(strings.Repeat("b", blockSize-headerSize)),
-		[]byte(strings.Repeat("c", blockSize-headerSize)),
+// generateTestRecords returns up to 256 records in a slice of slices of the specified lengths.
```

Comment:
Okay, fixed the comment. I tried byte(258) and that failed, but i := 258; byte(i) works fine.